doc/service_reference_card.md

```diff
@@ -0,0 +1,56 @@
+# Service Reference Card Template
+
+* Daemons running
+  * httpd
+  * crond
+  * atd
+  * apeldbloader-cloud
+
+* Init commands (services started on container creation)
+  * service httpd start
+  * service crond start
+  * service atd start
+  * service apeldbloader-cloud start
+
+* Configuration files location
+  * /etc/httpd/conf.d/apel_rest_api.conf - forces SSL
+  * /etc/httpd/conf.d/ssl.conf - configures SSL
+  * /etc/apel/cloudloader.cfg - configuration for the loader
+  * /etc/apel/cloudsummariser.cfg - configuration for the summariser
+
+* Logfile locations
+  * /var/log/cloud/loader.log - log file for the loader
+  * /var/log/cloud/summariser.log - log file for the summariser
+
+* Open ports
```

Comment:
Language in this section still seems a bit clunky (from the admin doc).

Maybe reword the second half of the port 80 and 443 points to something like "the Apache server [/for HTTPS] forwards traffic to the APEL server for a particular URL pattern [true?]."